NodeOpera/generate.sh

```diff
@@ -7,6 +7,9 @@ echo "# !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!" > ./Dockerfile
 echo "# NOTE: DO *NOT* EDIT THIS FILE.  IT IS GENERATED." >> ./Dockerfile
 echo "# PLEASE UPDATE Dockerfile.txt INSTEAD OF THIS FILE" >> ./Dockerfile
 echo "# !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!" >> ./Dockerfile
+echo "FROM ubuntu:bionic AS builder-codecs-ffmpeg" >> ./Dockerfile
+echo "RUN apt-get update -qqy && apt-get -qqy install chromium-codecs-ffmpeg-extra" >> ./Dockerfile
+echo "" >> ./Dockerfile
```

Comment:
Why don't you add these lines to the `NodeOpera/Dockerfile.txt` right away?

Reply:
Because theses lines need to be before `echo FROM ${NAMESPACE}/node-base:${VERSION} >> ./Dockerfile`